src/pkg/packager/create.go

```diff
@@ -10,34 +10,43 @@ import (
 
 	"github.com/defenseunicorns/zarf/src/config"
 	"github.com/defenseunicorns/zarf/src/internal/packager/validate"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/packager/creator"
 )
 
 // Create generates a Zarf package tarball for a given PackageConfig and optional base directory.
 func (p *Packager) Create() (err error) {
-
 	cwd, err := os.Getwd()
 	if err != nil {
 		return err
 	}
 
-	if err := p.cdToBaseDir(p.cfg.CreateOpts.BaseDir, cwd); err != nil {
-		return err
+	if err := os.Chdir(p.cfg.CreateOpts.BaseDir); err != nil {
+		return fmt.Errorf("unable to access directory %q: %w", p.cfg.CreateOpts.BaseDir, err)
 	}
 
-	if err := p.load(); err != nil {
+	message.Note(fmt.Sprintf("Using build directory %s", p.cfg.CreateOpts.BaseDir))
+
+	pc := creator.NewPackageCreator(p.cfg.CreateOpts, &p.cfg.PkgOpts, p.cfg, cwd)
+
+	loadedPkg, warnings, err := pc.LoadPackageDefinition(p.layout)
```

Comment:
```suggestion
	var warnings []string
	p.cfg.Pkg, warnings, err = pc.LoadPackageDefinition(p.layout)
```

can do reassignment instead of intermediary variables